pkgs/top-level/all-packages.nix

```diff
@@ -9295,6 +9295,8 @@ with pkgs;
 
   memtester = callPackage ../tools/system/memtester { };
 
+  meraki = python3Packages.callPackage ../development/python-modules/meraki { };
```

Comment:
Put this in `pkgs/top-level/python-modules.nix` instead. If you really want it at the top-level pkgs (e.g. if it provides a CLI), than you can `inherit (python3Packages) meraki;` here after adding it to the other file

Reply:
There is a separate package that provides a CLI that I'm working on packaging. It needs this library to be packaged first though.